io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "io.github.brunoherbelin.Vimix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "vimix",
+    "rename-desktop-file": "vimix.desktop",
+    "rename-icon": "vimix",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--device=shm",
+        "--device=all",
+        "--filesystem=host",
```

Comment:
host filesystem is broad.

Reply:
ok, to be discussed ; i want the user to be able to:
- read files in the account and mounted drives
- access `/dev/input0` - and others (v4l2)
- access `/tmp` : needed explicitly for shared memory